src/server.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+require('dotenv').config()
+const express = require('express')
+const cors = require('cors');
+const app = express()
+
+const routes = require('./routes')
+const notFoundHandler = require('./handlers/404')
+const serverErrorHandler = require('./handlers/500')
+const corsOptions = {
+  origin:process.env.CORS_ORIGIN, 
+  credentials:true,
+  optionSuccessStatus:200
+}
+
+app.use(cors(corsOptions));
+app.use(express.json())
+app.use(routes)
+app.use('*',notFoundHandler)
+app.use(serverErrorHandler)
+
+module.exports = {
+  app,
+  start: port => {
+    console.log('@@',port)
```

Comment:
lets remove this comment and fall back to port 3000 instead of erroring.

Reply:
removed the logs from server.js and updated routes.js to use arrow function syntax